scripts/add_translation.py

```diff
@@ -0,0 +1,160 @@
+
+""" Download all files from a selected google drive in csv format then convert to a json format. The script
+    can repeat this process upon user request.
+
+    Requires access to the main google drive folder (or credentials from a developer who
+    has access). The ID for the main google folder is stored in TRANSLATION_FOLDER_ID.
+
+    Steps:
+    1. Obtain client_secret.json file:
+       a. Go to https://console.developers.google.com/
+       b. Create a project for ISP
+       c. Under the credentials tab, click Create Credentials --> OAuth client ID, select "Other"
+          as application type and click "Create"
+       d. Find the newly created credentials under "OAuth 2.0 client IDs" and download as json
+       e. Rename json file to "client_secret.json"
+    2. In the scripts directory, create a 'credentials' folder and add the client_secret.json file
+    3. Run the script.
+    4. Move the content of the generated files as described in the Readme.md file.
+
+
+    The first time you run this script, you may get a message saying that you need to authorize specific APIs for use
+    with this project. The message will provide instructions needed to complete this process; wait several minutes
+    and then try running again.
+
+    Modifies: https://developers.google.com/drive/v3/web/quickstart/python """
+
+import httplib2
+import os
+import sys
+
+import consent_form_json
+
+from apiclient import discovery
+from oauth2client import file, client, tools
+
+try:
+    import argparse
+
+    flags = argparse.ArgumentParser(parents=[tools.argparser]).parse_args()
+except ImportError:
+    flags = None
+
+# If modifying these scopes, delete your previously saved credentials
+# at isp/scripts/credentials/credentials.json
+SCOPES = 'https://www.googleapis.com/auth/drive'
+CLIENT_SECRET_FILE = 'credentials/client_secret.json'
+APPLICATION_NAME = 'International Situations Project'
+
+# Main google directory that contain all translation folders
+TRANSLATION_FOLDER_ID = '0B441UYO1vv_CVjRrc25SZjRhazA'
+
+# Downloaded csv files with each site's consent form
+files_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'consent_forms')
+
+# List of languages with their IDs
+LANGUAGE_LIST = {}
```

Comment:
Please avoid using global variables, in any python function- shared state can be very hard to debug.

It's ok to use a list of constants when that list doesn't change (like EXCLUDED_ATTRIBUTES)- but as soon as global variables are being modified from inside functions, things can get weird.

Instead, consider communicating between functions by breaking logic into manageable chunks and passing variables. For example...

```
language_list = get_language_list()
language_list = sorted(language_list)  # Can specify key argument to dictate what field is used for sort, etc
choice = display_and_pick_language(language_list)
do_translation_mojo(choice)
```